app/src/main/res/values/config-common.xml

```diff
@@ -37,6 +37,11 @@
     <integer name="config_accessibility_long_press_key_timeout">3000</integer>
     <integer name="config_max_popup_keys_column">5</integer>
 
+    <integer name="config_default_gesture_fast_typing_cooldown">350</integer>
```

Comment:
The current default value seems to be `config_gesture_static_time_threshold_after_fast_typing` (via attr `gestureStaticTimeThresholdAfterFastTyping`), wouldn't it be more consistent to re-use it?

Reply:
That's a great question, I was pretty confused by that as well. Yes, there's that attribute you pointed out. But what actually seems to be used in practice for this particular cooldown is defined [here](https://github.com/Helium314/HeliBoard/blob/main/app/src/main/java/helium314/keyboard/keyboard/internal/GestureStrokeRecognitionParams.java#L43) as 350.

I don't know why those are separate values, perhaps they do different things? This last commit voids `mStaticTimeThresholdAfterFastTyping` for that particular cooldown, but the variable is also used here:
https://github.com/Helium314/HeliBoard/blob/4aac81391e9e505724a315585d4de13b55767c0a/app/src/main/java/helium314/keyboard/keyboard/PointerTracker.java#L158-L160

Where I think the value is subsequently used for bogus-detection stuff that I don't want my setting to affect. But I don't know which value is used in that case, pre-change.

Reply:
> But what actually seems to be used in practice for this particular cooldown is defined [here](https://github.com/Helium314/HeliBoard/blob/main/app/src/main/java/helium314/keyboard/keyboard/internal/GestureStrokeRecognitionParams.java#L43) as 350.

Did you check whether the 350 is really used in app? Looking at [`TypedArray.getInt`](https://developer.android.com/reference/android/content/res/TypedArray#getInt(int,%20int)) this should just be the default value in case the attribute is not present in `mainKeyboardViewAttr`.
But if it's always the 350, then let's also use that as default.

Reply:
Okay yeah, it looks like it's always 500 in vanilla and that 350 would be used if the 500 weren't defined in `mainKeyboardViewAttr`.

The 350 is an oddity. Maybe someone tried to reduce that default threshold from 500 a while ago but didn't realize it needed to be modified elsewhere? But I guess the default should be 500 since that's what it's been in practice.